DB/src/main/java/io/deephaven/db/tables/utils/ParquetTools.java

```diff
@@ -379,36 +380,9 @@ private static Table readTableFromSingleParquetFile(
                 RegionedTableComponentFactoryImpl.INSTANCE, locationProvider, null);
     }
 
-    private static Class<?> dbArrayType(final Class<?> componentTypeFromParquet) {
-        if (componentTypeFromParquet == null) {
-            return null;
-        }
-        if (componentTypeFromParquet.equals(int.class)) {
-            return DbIntArray.class;
-        }
-        if (componentTypeFromParquet.equals(long.class)) {
-            return DbLongArray.class;
-        }
-        if (componentTypeFromParquet.equals(byte.class)) {
-            return DbByteArray.class;
-        }
-        if (componentTypeFromParquet.equals(double.class)) {
-            return DbDoubleArray.class;
-        }
-        if (componentTypeFromParquet.equals(float.class)) {
-            return DbFloatArray.class;
-        }
-        if (componentTypeFromParquet.equals(char.class)) {
-            return DbCharArray.class;
-        }
-        if (componentTypeFromParquet.equals(short.class)) {
-            return DbShortArray.class;
-        }
-        if (componentTypeFromParquet.equals(Boolean.class)) {
-            return DbArray.class;
-        }
-        return null;
-    }
+    private static final SimpleTypeMap<Class<?>> dbArrayTypeMap = SimpleTypeMap.create(
```

Comment:
CONSTANTS_SHOULD_LOOK_LIKE_THIS ;-)
Where's `byte`?
For reference, I had:
```
    private static final SimpleTypeMap<Class<? extends DbArrayBase>> COMPONENT_TYPE_TO_DBARRAY_TYPE = SimpleTypeMap.create(
            DbArray.class, DbCharArray.class, DbByteArray.class, DbShortArray.class, DbIntArray.class, DbLongArray.class, DbFloatArray.class, DbDoubleArray.class, DbArray.class);
```
I see that you don't need a mapping for `boolean`.

Reply:
And maybe fix `.equals` to `==` for classes in this file.